torch/_inductor/codegen/common.py

```diff
@@ -72,6 +72,27 @@ def _print_CleanDiv(self, expr):
         return self._print_FloorDiv(expr)
 
 
+class PythonPrinter(ExprPrinter):
+    def _print_ModularIndexing(self, expr):
+        x, div, mod = expr.args
+        x = self.paren(self.doprint(x))
+        div = self.paren(self.doprint(div))
+        mod = self.paren(self.doprint(mod))
+        if div != "1":
+            x = f"({x} // {div})"
+        return f"{x} % {mod}"
+
+    def _print_FloorDiv(self, expr):
+        x, div = expr.args
+        x = self.paren(self.doprint(x))
+        div = self.paren(self.doprint(div))
+        return f"({x} // {div})"
```

Comment:
Is the outer parentheses here necessary?

Reply:
Probably not, but it's unchanged from existing.